src/Files.App/Actions/Content/Background/SetAsAppBackgroundAction.cs

```diff
@@ -18,12 +18,11 @@ public override RichGlyph Glyph
 
 		public override bool IsExecutable =>
 			base.IsExecutable &&
-			context.SelectedItem is not null;
+			ContentPageContext.SelectedItem is not null;
 
 		public override Task ExecuteAsync(object? parameter = null)
 		{
-			if (context.SelectedItem is not null)
```

Comment:
@hishitetsu can you confirm if this is okay to remove? I'm pretty sure it's needed to prevent a crash.

Reply:
Yeah, this code is necessary because the item may be deselected between the time it is checked in `IsExecutable` and the time the action is actually executed.

Reply:
Why not use IsExecutable prop instead?

Reply:
Certainly, but I noticed we should do the following instead of simply checking to prevent NullReferenceException completely.
```
if (ContentPageContext.SelectedItem is ListedItem selectedItem)
	AppearanceSettingsService.AppThemeBackgroundImageSource = selectedItem.ItemPath;
```

Reply:
Done.